<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dates</class>
 <widget class="QMainWindow" name="Dates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date Picker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>149</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select a date range</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="StartDate">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="EndDate">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>Dates</receiver>
   <slot>dateEffector(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dates</receiver>
   <slot>submitDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dateEffector(QDate)</slot>
  <slot>submitDate()</slot>
 </slots>
</ui>
